pkg/models/sigv4.go

```diff
@@ -97,6 +97,8 @@ func (m *SigV4Middleware) signer() (*v4.Signer, error) {
 		c = credentials.NewStaticCredentials(m.Config.AccessKey, m.Config.SecretKey, "")
 	case Credentials:
 		c = credentials.NewSharedCredentials("", m.Config.Profile)
+	default:
+		panic(fmt.Sprintf("Unexpected auth type %q", authType))
```

Comment:
I think it should not panic here in case the [default](https://github.com/grafana/grafana/blob/bdbe127a304ee5f0f4298a2a8ba08b09e97515b8/pkg/models/sigv4.go#L21) authentication is selected.
This case used to be handled [here](https://github.com/grafana/grafana/blob/bdbe127a304ee5f0f4298a2a8ba08b09e97515b8/pkg/models/sigv4.go#L105) (no credentials are set).

Reply:
@papagian Thanks, I will double check.

Reply:
@papagian Ah, I didn't notice that check. I guess it goes to show that explicit is better than implicit :)